QEfficient/generation/text_generation_inference.py

```diff
@@ -342,6 +345,16 @@ def __init__(
             full_batch_size if full_batch_size else self._fetch_full_batch_size()
         )  # Check and fetch full batch size if CB is enabled
 
+        if prompt_to_lora_id_mapping:
```

Comment:
Thanks for the suggestion. I added this line to the `generate()` function in `lora/auto.py` so that the user can directly modifies the list they passed in. I set `prompt_to_adapter_mapping` to default to ‘base’ if not provided. This way, `lora_id` will default to 0 and use the base model.